tests/string_tensor_test.go

```diff
@@ -8,6 +8,21 @@ import (
 	"gorgonia.org/tensor"
 )
 
+// uses test struct from mnist_cnn_test
+func generate_string_tests() []test {
```

Comment:
Is this the same model that I shared with you or a different one? Can you provide an input shape of [1, n]?

Reply:
This is the model you provided. I did them one by one so they would be separate inferences but I'll include a test case where there is more than one.

Reply:
Gotcha! This model does take [1,n] input shape so you should be able to provide it.